frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunList.tsx

```diff
@@ -25,7 +26,15 @@ export const PipelineRunList: React.FC<PipelineRunListProps> = (props) => {
     'pipelines',
   );
   const [taskRuns, taskRunsLoaded] = useTaskRuns(props.namespace);
-
+  const [TRtaskRuns, TRtaskRunsLoaded] = useTRTaskRuns(props.namespace);
+  const mergedTaskRuns =
+    TRtaskRunsLoaded || taskRunsLoaded
+      ? taskRuns.concat(
+          TRtaskRuns.filter(
+            (plr) => !taskRuns.some((k8sPlr) => k8sPlr.metadata.uid === plr.metadata.uid),
```

Comment:
This will iterate over all k8s resources, for each result-PLR. Please extract the k8s uid first into a Set and remove the duplicates then with [set.has](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has)